DOCUMENTATION.md

```diff
@@ -1,810 +1,807 @@
 
-[![Gem Version](https://badge.fury.io/rb/notifications-ruby-client.svg)](https://badge.fury.io/rb/notifications-ruby-client)
+# Set up the client
 
-## Table of Contents
+## Install the client
 
-* [Installation](#installation)
-* [Getting started](#getting-started)
-* [Send messages](#send-messages)
-* [Get the status of one message](#get-the-status-of-one-message)
-* [Get the status of all messages](#get-the-status-of-all-messages)
-* [Get a template by ID](#get-a-template-by-id)
-* [Get a template by ID and version](#get-a-template-by-id-and-version)
-* [Get all templates](#get-all-templates)
-* [Generate a preview template](#generate-a-preview-template)
-* [Get received texts](#get-received-texts)
-
-## Installation
-
-Prior to usage an account must be created through the Notify admin console. This will allow access to the API credentials you application.
-
-You can then install the gem or require it in your application.
+Run the following code in the command line:
 
 ```
 gem install 'notifications-ruby-client'
 ```
 
-## Getting started
+Refer to the [client changelog](https://github.com/alphagov/notifications-ruby-client/blob/master/CHANGELOG.md) for the version number and the latest updates.
+
+## Create a new instance of the client
+
+Add this code to your application:
 
 ```ruby
 require 'notifications/client'
-client = Notifications::Client.new(api_key)
+client: Notifications::Client.new(api_key)
 ```
 
-Generate an API key by logging in to GOV.UK Notify [GOV.UK Notify](https://www.notifications.service.gov.uk) and going to the **API integration** page.
+To get an API key, [sign in to GOV.UK Notify](https://www.notifications.service.gov.uk/) and go to the __API integration__ page. You can find more information in the [API keys](/#api-keys) section of the documentation.
 
-## Send messages
+# Send a message
 
-### Text message
+You can use GOV.UK Notify to send text messages, emails and letters.
 
-#### Method
+## Send a text message
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+### Method
 
 ```ruby
-sms = client.send_sms(
-  phone_number: number,
-  template_id: template_id,
-  personalisation: {
-    name: "name",
-    year: "2016",                      
-  },
-  reference: "your_reference_string",
-  sms_sender_id: sms_sender_id
-) # => Notifications::Client::ResponseNotification
+sms: client.send_sms(
+  phone_number: "+447900900123",
+  template_id: "f33517ff-2a88-4f6e-b855-c550268ce08a",
+)
 ```
 
-</details>
+### Arguments
 
-#### Response
+#### phone_number (required)
 
-If the request is successful, a `Notifications::Client:ResponseNotification` is returned.
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+The phone number of the recipient of the text message. This number can be a UK or international number.
 
 ```ruby
-sms => Notifications::Client::ResponseNotification
+phone_number:"+447900900123",
```

Comment:
I would leave out the code example here and for `template_id` below - since they are both required arguments you can see what they look like a few lines above in the method section, so it saves repetition.

Reply:
I need to make a decision on this - Python doesnt have the examples but Java does